MonoGame.Framework/Input/GamePad.tvOS.cs

```diff
@@ -18,10 +18,15 @@ private static int PlatformGetMaxIndex()
             return 4;
         }
 
+        private static int PlatformGetMaxNumberOfGamePads ()
+        {
+            return PlatformGetMaxIndex ();
+        }
```

Comment:
I think you can just remove the `PlatformGetMaxIndex()` function entirely and return 4 here.